fsspec/tests/test_core.py

```diff
@@ -21,7 +21,9 @@
 
 
 @contextmanager
-def tempzip(data={}):
+def tempzip(data=None):
+    if data is None:
+        data = {}
```

Comment:
I usually write 
```suggestion
    data = data or {}
```

Reply:
I prefer it too, but other maintainers find it less readable. Will change it.